doc/configuration.rst

```diff
@@ -406,36 +406,89 @@ configuration option setup for Sphinx-Gallery.
     :emphasize-lines: 12-22, 32-42
     :linenos:
 
-.. note::
-   By default, Sphinx-gallery will inspect global variables (and code objects)
-   at the end of each code block to try to find classes of variables and
-   method calls. It also tries to find methods called on classes.
-   For example, this code::
-
-       lst = [1, 2]
-       fig, ax = plt.subplots()
-       ax.plot(lst)
-
-   should end up with the following links (assuming intersphinx is set up
-   properly):
-
-   - :class:`lst <python:list>`
-   - :func:`plt.subplots <matplotlib.pyplot.subplots>`
-   - :class:`fig  <matplotlib.figure.Figure>`
-   - :class:`ax <matplotlib.axes.Axes>`
-   - :meth:`ax.plot <matplotlib.axes.Axes.plot>`
-
-   However, this feature is might not work properly in all instances.
-   Moreover, if variable names get reused in the same script to refer to
-   different classes, it will break.
-
-   To disable this global variable introspection, you can use the configuration
-   key::
-
-       sphinx_gallery_conf = {
-           ...
-           'inspect_global_variables'  : False,
-        }
+Toggling global variable inspection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, Sphinx-gallery will inspect global variables (and code objects)
+at the end of each code block to try to find classes of variables and
+method calls. It also tries to find methods called on classes.
+For example, this code::
+
+    lst = [1, 2]
+    fig, ax = plt.subplots()
+    ax.plot(lst)
+
+should end up with the following links (assuming intersphinx is set up
+properly):
+
+- :class:`lst <python:list>`
+- :func:`plt.subplots <matplotlib.pyplot.subplots>`
+- :class:`fig  <matplotlib.figure.Figure>`
+- :class:`ax <matplotlib.axes.Axes>`
+- :meth:`ax.plot <matplotlib.axes.Axes.plot>`
+
+However, this feature is might not work properly in all instances.
```

Comment:
this feature might 
(I know it was not introduced here, but while you're at it :-))